mobile/src/bundles/common/enums/styles/styles.enum.ts

```diff
@@ -0,0 +1,9 @@
+const AlignmentValues = {
+    CENTER: 'center',
+    FLEX_START: 'flex-start',
+    FLEX_END: 'flex-end',
+    STRETCH: 'stretch',
+    SPACE_BETWEEN: 'space-between',
+} as const;
```

Comment:
```suggestion
const AlignmentValue = {
    CENTER: 'center',
    FLEX_START: 'flex-start',
    FLEX_END: 'flex-end',
    STRETCH: 'stretch',
    SPACE_BETWEEN: 'space-between',
} as const;
```

Reply:
I transferred it to the flex.ts file. But I made constants for the values because of the error "using a value instead of a constant more than 3 times".